pyshipping/fortras/entl.py

```diff
@@ -188,18 +188,18 @@ def parse(self, data):
             if line[0] == 'M':
                 match = re.search(Entladebericht.m_record_re, line)
                 if not match:
-                    print 'no match', repr(line)
+                    print('no match', repr(line))
                 # the content of M records are ignored
             elif line[0] == 'N':
                 match = re.search(Entladebericht.n_record_re, line)
                 if not match:
-                    print 'no match', repr(line)
+                    print('no match', repr(line))
                 # the content of N records are ignored
             elif line[0] == 'V':
                 match = re.search(Entladebericht.v_record_re, line)
                 newdict = {}
                 if not match:
-                    print 'no match', repr(line)
+                    ('no match', repr(line))
```

Comment:
Why did you delete this print?

Reply:
Well spotted, I think that one was deleted in eror.